### ISSUE TITLE
Failed to pull image "swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:v0.7.0-111-g8effddd":

### ISSUE THREAD

**Issue by magnate3** [karmada-io/karmada#621]
**What happened**:

Failed to pull image "swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:v0.7.0-111-g8effddd": rpc error: code = NotFound desc = failed to pull and unpack image "swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:v0.7.0-111-g8effddd": failed to resolve reference "swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:v0.7.0-111-g8effddd": swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:v0.7.0-111-g8effddd: not found

I solve the problem by the following way:
firstly,create your privte  docker images registry
secondly,  excute cmd:   env REGISTRY="your  docker images registry"  make upload-images and 
env REGISTRY="your  docker images registry" make image-karmada-agent
thirdly, change the referred images of such files :  artifacts/agent/karmada-agent.yaml、artifacts/deploy/controller-manager.yaml、artifacts/deploy/karmada-scheduler.yaml、artifacts/deploy/karmada-webhook.yaml

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Karmada version:
- Others:

**Comment by RainbowMango** [comment]
We don't have the `v0.x.x-xxx-g8effddd` images on the public registry.  Have you changed the version in `artifacts/deploy/controller-manager.yaml`?

**Comment by magnate3** [comment]
no, I deploy  karmada in my arm64 cluster, and many docker images need to be builded by myself.
so,I use myself images tag.
could you tell me what    public registry you use?

**Comment by RainbowMango** [comment]
> swr.ap-southeast-1.myhuaweicloud.com/karmada

It is `swr.ap-southeast-1.myhuaweicloud.com/karmada`.

> no, I deploy karmada in my arm64 cluster, and many docker images need to be builded by myself.

Good to know. Maybe we can publish arm64 images too.

**Comment by magnate3** [comment]
@RainbowMango  in a short ,i use my prive registry and the problem solved  in arm64 k8s
